contracts/CoveragePool.sol

```diff
@@ -68,11 +68,16 @@ contract CoveragePool is Ownable {
     }
 
     /// @notice Begins risk manager approval process.
-    /// @dev Can be called only by the contract owner. For a risk manager to be
+    /// @dev Can be called only by the contract owner and only when the first
+    ///      risk manager is already approved. For a risk manager to be
     ///      approved, a call to `finalizeRiskManagerApproval` must follow
     ///      (after a governance delay).
     /// @param riskManager Risk manager that will be approved
     function beginRiskManagerApproval(address riskManager) external onlyOwner {
+        require(
+            firstRiskManagerApproved,
+            "First risk manager was not approved"
```

Comment:
Nitpicking but let's make the revert message more precise. This function is for approving risk manager and we are currently saying it cannot be executed because no risk manager was yet approved. I suggest something like:
```
The first risk manager is not yet approved; Please use  approveFirstRiskManager instead
```

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/176/commits/3aa4ecce94e64166f1f4fece8d634156dab0ef1c.